jac-uk/admin
# Feat/2254 streamlined publication of external adverts

## What's included?
Closes #2254 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to the preview link: https://jac-admin-develop--pr2301-feat-2254-streamline-0we56wtz.web.app/exercises
- Create external vacancy:
    - click `Create an exercise`
<img width="1512" alt="Screenshot 2024-02-06 at 16 39 33" src="https://github.com/jac-uk/admin/assets/156695133/843f3b0c-6594-4726-9dcf-fd9f5e602eb5">

    - Choose `No - this is an external vacancy` and save.

<img width="886" alt="Screenshot 2024-02-06 at 16 41 06" src="https://github.com/jac-uk/admin/assets/156695133/bf44cc61-688b-4878-bee7-e7432283c3be">

    - It should create an draft external vacancy and show overview of the vacancy.
- Edit external vacancy
    - Go to vacancy overview. (Click the exercise, if you're in the exercise list https://jac-admin-develop--pr2301-feat-2254-streamline-0we56wtz.web.app/exercises)
    - Click `update`, it should show edit page.
    
<img width="1051" alt="Screenshot 2024-02-06 at 16 44 41" src="https://github.com/jac-uk/admin/assets/156695133/459b9cb3-20f0-49fa-83a5-1b697212c1dc">

    - Enter data and click save, if should save those data.
- Publish external vacancy
    - Go to vacancy overview. (Click the exercise, if you're in the exercise list https://jac-admin-develop--pr2301-feat-2254-streamline-0we56wtz.web.app/exercises)
    - Scroll down, there's a `Publish on website` button under the files downloads.
   
<img width="1499" alt="Screenshot 2024-02-06 at 16 51 13" src="https://github.com/jac-uk/admin/assets/156695133/cb060b69-5bd0-4519-8839-965f84c33677">

    - Click the `Publish on website`, it should be posted on the apply site.
   

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, video demo, notes etc.

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
